server/fleet/queries.go

```diff
@@ -97,16 +96,6 @@ func verifyQuerySQL(query string) error {
 	if emptyString(query) {
 		return errQueryEmptyQuery
 	}
-	if err := verifySQL(query); err != nil {
-		return err
-	}
-	return nil
-}
-
-func verifySQL(query string) error {
-	if validateSQLRegexp.MatchString(query) {
```

Comment:
Looks like `validateSQLRegexp` is also used by `verifyPolicyQuery` in `server/fleet/policies.go`, should that be removed too? And then `validateSQLRegexp` can be deleted?

Reply:
Oh good catch. Yes. Removing.

Reply:
[a55a2a8](https://github.com/fleetdm/fleet/pull/10427/commits/a55a2a81b6ef2fe262229287f5e78f7e42b1164f)